phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,62 @@
-<!-- INCLUDE ucp_header.html -->
-
-<!-- IF not PROMPT -->
-	<!-- INCLUDE ucp_pm_message_header.html -->
-<!-- ENDIF -->
-
-<!-- IF PROMPT -->
-	<h2>{L_EXPORT_AS_CSV}</h2>
-	<form id="viewfolder" method="post" action="{S_PM_ACTION}">
-	<div class="panel">
-		<div class="inner">
-		<h3>{L_OPTIONS}</h3>
-		<fieldset>
-			<dl>
-				<dt><label for="delimiter">{L_DELIMITER}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="delimiter" name="delimiter" value="," /></dd>
-			</dl>
-			<dl>
-				<dt><label for="enclosure">{L_ENCLOSURE}{L_COLON}</label></dt>
-				<dd><input class="inputbox" type="text" id="enclosure" name="enclosure" value="&#034;" /></dd>
-			</dl>
-		</fieldset>
-		</div>
-	</div>
-	<fieldset class="submit-buttons">
-		<input type="hidden" name="export_option" value="CSV" />
-		<input class="button1 button button-form" type="submit" name="submit_export" value="{L_EXPORT_FOLDER}" />&nbsp;
-		<input class="button1 button button-form-bold" type="reset" value="{L_RESET}" name="reset" />&nbsp;
-		{S_FORM_TOKEN}
-	</fieldset>
-	</form>
-
-<!-- ELSE -->
-
-	<!-- IF NUM_REMOVED -->
-		<div class="notice">
-			<p>{RULE_REMOVED_MESSAGES}</p>
-		</div>
-	<!-- ENDIF -->
-
-	<!-- IF NUM_NOT_MOVED -->
-		<div class="notice">
-			<p>{NOT_MOVED_MESSAGES}<br />{RELEASE_MESSAGE_INFO}</p>
+{% include 'ucp_header.html' %}
+
+{% if not PROMPT %}
+	{% include 'ucp_pm_message_header.html' %}
+{% endif %}
+
+{% if messagerow %}
+	<div class="pm-main">
+		<div class="pm-left-panel">
+			<ul class="pm-left-primary">
+				{% for message in messagerow %}
+					<li class="message-row {{ message.S_ROW_COUNT is odd ? 'row1' : 'row2' }} {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'msg-unread' }}">
+						<span class="left-inner">
+
+							<a href="{message.U_VIEW_PM}" class="msg-title {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'title-unread' : 'title-read' }}">{message.SUBJECT}</a><br><br>
+							<span class="pm-left-secondary">
+								<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous">
+
+								{% if S_SHOW_RECIPIENTS %}
+								<span class="row-msg-author">
+									{{ lang('MESSAGE_TO') }} {message.RECIPIENTS}
+								{% else %}
+									{message.MESSAGE_AUTHOR_FULL}
+								</span>
+								<span class="row-time-date">
+									<i class="far fa-clock fa-fw" aria-hidden="true"></i>
+									{message.SENT_TIME}
+								</span>
+								{% endif %}
```

Comment:
This `endif` should be on line 25. Sent time should not be influenced by `S_SHOW_RECIPIENTS` variable.